source/plugin/bookview.rst

```diff
@@ -0,0 +1,36 @@
+==========
+Book Views
+==========
+
+A ``BookView`` is the displaying of the Book GUI on the client. The ``BookView`` is not associated with an actual
+``ItemStack`` and is only for displaying ``Text`` through a book to the player. Note that a ``BookView`` is read-only,
+due to it being impossible to tell the client to open an unsigned book.
+
+To create a ``BookView``, we simply need to obtain a ``BookView.Builder``, which is provided through the
+``BookView#builder()`` method. Using the provided builder, we can specify the title of the ``BookView``, the author,
```

Comment:
You use `BookView` twice in this sentence. It would be better to say "Using the builder, we can specify the title, author, and pages of the `BookView`."